src/rbac/domain/project/entity/project_role.py

```
@@ -0,0 +1,40 @@
+class ProjectRole(str, Enum):
+    # 변수명 을 그대로 value로 사용하기 위해 auto() 메서드를 오버라이드
+    # 주의 : 변수명보다 앞서 오버라이드 되어야 함
+    def _generate_next_value_(name, start, count, last_values):
+        return name  # 변수명을 문자열 그대로 value로 사용
+
+    VIEWER = auto()
+    EDITOR = auto()
+    PROJECT_OWNER = auto()
+
+    @classmethod
+    def from_str(cls, value: str) -> "AccountRole":
```

Ruff (0.8.2)

Undefined name `AccountRole`

(F821)